docs/tasks.md

```diff
@@ -6,105 +6,82 @@ weight: 1
+  - [`apiVersion`][kubernetes-overview] - Specifies the API version. For example, 
     `tekton.dev/v1beta`.
-  - [`kind`][kubernetes-overview] - Specify the `Task` resource object.
-  - [`metadata`][kubernetes-overview] - Specifies data to uniquely identify the
-    `Task` resource object, for example a `name`.
+  - [`kind`][kubernetes-overview] - Identifies this resource object as a `Task` object.
+  - [`metadata`][kubernetes-overview] - Specifies metadata that uniquely identifies the
+    `Task` resource object. For example, a `name`.
   - [`spec`][kubernetes-overview] - Specifies the configuration information for
-    your `Task` resource object. `Task` steps must be defined through either of
-    the following fields:
-    - [`steps`](#steps) - Specifies one or more container images that you want
-      to run in your `Task`.
+    this `Task` resource object. 
+  - [`steps`](#defining-steps) - Specifies one or more container images to run in the `Task`.
 - Optional:
-  - [`description`](#description) - Description of the Task.
-  - [`params`](#parameters) - Specifies parameters
-  - [`resources`](#resources) - Specifies
+  - [`description`](#adding-a-description) - An informative description of the `Task`.
+  - [`params`](#parameters) - Specifies execution parameters for the `Task`.
+  - [`resources`](#resources) - **alpha only** Specifies
     [`PipelineResources`](resources.md) needed or created by your
-    `Task`. *Note: this is an alpha field, it is not supported as the
-    rest of the beta field*.
-    - [`inputs`](#inputs-resources) - resources needed by your `Task`.
-    - [`outputs`](#outputs-resources) - resources created by your `Task`
-  - [`workspaces`](#workspaces) - Specifies paths at which you expect volumes to
-    be mounted and available
-  - [`results`](#results) - Specifies the result file name where the task can write its result
-  - [`volumes`](#volumes) - Specifies one or more volumes that you want to make
-    available to your `Task`'s steps.
-  - [`stepTemplate`](#step-template) - Specifies a `Container` step
-    definition to use as the basis for all steps within your `Task`.
-  - [`sidecars`](#sidecars) - Specifies sidecar containers to run alongside
-    steps.
+    `Task`.
+    - [`inputs`](#inputs-resources) - Specifies the resources ingested by the `Task`.
+    - [`outputs`](#outputs-resources) - Specifies the resources produced by the `Task`.
+  - [`workspaces`](#workspaces) - Specifies paths to volumes required by the `Task`.
+  - [`results`](#results) - Specifies the file to which the `Tasks` writes its execution results.
+  - [`volumes`](#volumes) - Specifies one or more volumes that will be available available to the `Steps` in the `Task`.
+  - [`stepTemplate`](#step-template) - Specifies a `Container` step definition to use as the basis for all `Steps` in the `Task`.
+  - [`sidecars`](#sidecars) - Specifies `Sidecar` containers to run alongside the `Steps` in the `Task.
```

Comment:
i think you changed the section names for a lot of the above so the links dont work anymore

Reply:
That's exactly what happened... I just went over all the links and updated them. Should be good now.